tests/integration/targets/support_tunnel/tasks/main.yml

```diff
@@ -0,0 +1,89 @@
+---
+- environment:
+    SC_HOST: "{{ sc_host }}"
+    SC_USERNAME: "{{ sc_username }}"
+    SC_PASSWORD: "{{ sc_password }}"
+    SC_TIMEOUT: "{{ sc_timeout }}"
+
+  block:
+    - name: Cleanup
```

Comment:
It looks like returned `tunnel_open` is sometimes int, sometimes boolean. Can we return the same datatype always? We can also say it is not number, but a string, and than "" is a special value meaning tunnel is not closed, and non-empty value means tunnel is opened. Class `SupportTunnel` has `__init__(tunnel_open: str` too.

Reply:
We could also add `open: bool` as one of the output parameters. That way we make it easier for testing (True/False) and less ambiguous for user. If you both agree @justinc1 @PolonaM .

Reply:
@domendobnikar I agree with the proposal, so - we have code: int and open: bool